stable/weave-cloud/templates/manifest.yaml

```diff
@@ -0,0 +1,402 @@
+apiVersion: v1
+kind: List
+items:
+  - metadata:
+      labels:
+        name: weave-flux
+      name: weave-flux
+      annotations:
+        cloud.weave.works/launcher-info: |-
```

Comment:
These are produced by our generator API, and are designed to help us supporting our users, so we can tell exactly when user installed the resources and how far are they behind latest. I am happy to modify the content of this annotation for the purpose of this chart.